src/CarBrands.ts

```diff
@@ -0,0 +1,4 @@
+export enum CarBrands {
```

Comment:
```suggestion
export enum CarBrand {
```

Reply:
I understand your suggestion and I'm fine making the change. However, I chose to make it plural because the `Regions` enum is also plural. For consistency I would suggest either we use plural for the enums or non-plural, but it's your call 🙂

Reply:
Lets stick to singular case, and the enums which are plural was a mistake on my end. Hopefully, will push a major version sometime and address those.

Reply:
Got it! I've renamed it to `CardBrand`.